spec/features/admin_orders_view_spec.rb

```diff
@@ -30,10 +31,10 @@
   end
 
   it "displays order date and time" do
-   within('table tr:nth-child(2)') do
+    within('table tr:nth-child(2)') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.